### PR TITLE
Cleanup of MBC1 docs

### PR DESCRIPTION
**Pull request by TheThief**

### CODE REVIEW

File: src/MBC1.md
Comment (TheThief) [inline src/MBC1.md:10]: https://github.com/gbdev/pandocs/pull/310#discussion_r669921583

TODO: Add a section for the large rom wiring so that the MMM01 docs can link to it

**Comment by ISSOtm** [comment]
I already begun a partial review some time ago, but I don't want to continue it before #317 is merged.

**Comment by ISSOtm** [comment]
Begun another partial review, but the mapping diagrams are basically blocked by #334.

**Comment by avivace** [comment]
Any update on this @ISSOtm ?

**Comment by ISSOtm** [comment]
I haven't worked on this (been busy on other PRs, as you know), but upstream has noted some fixes of their own, anyway.

**Review by avivace — APPROVED** [review]

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: src/MBC1.md
Comment (TheThief) [reply src/MBC1.md:10]: Common misconception, but no. The 2 bit register directly outputs on MBC pins, and is physically wired to ROM or RAM by the cart depending on which needs it.

**Review by avivace — APPROVED** [review]

**Pull request merged**
